File: problems/001_leetcode_id_509_fibonacci_number/javascript/approach_2.js

// Method 2: Recursive Approach

var fib = function(n) {
    if(n == 0) return 0
    if(n == 1) return 1
    return fib(n-1) + fib(n-2)
};

input = 4;
console.log(fib(input)); // Output: 3

File: problems/002_leetcode_id_1838_frequency_of_the_most_frequent_element/python/approach_2.py

# Method 2: Recursive Approach

def fib(n): 
    if(n == 0):
        return 0
    if(n == 1): 
        return 1
    return fib(n-1) + fib(n-2)

input = 4
print(fib(input)); # Output: 3